### ISSUE TITLE
Suggestions for alarm_control_panel-card

### ISSUE THREAD

**Issue by BoneheadFraggle** [ciotlosm/custom-lovelace#250]
I use wall mounted cell phones with https://github.com/thanksmister/wallpanel-android as wallpanels/dashboards, thermometers, camera, motion sensors and alarm keypads which works great.

For those using the alarm_control_panel-card as alarm keypad, it would be really nice to have a different approact to the keypad:

1: (Default view):
Only show the badge - and very large. No text at all. Then it would be easy to see the current status in a distance. You can see before getting close to the door if the alarm is activated or not.

2: If [Disarmed] 
When touching/clicking the badge the available badges are shown, arm_home and arm_away
Not until the desired one is clicked, the keypad is shown.

2: If [Armed]
Alt. 1 (when armed, you can only disarm): When touching/clicking the badge the keypad is shown
Alt 2: When touching/clicking the badge the available badges are shown, arm_home or arm_away + disarm. Not until the desired one is clicked, the keypad is shown.

**Comment by ciotlosm** [comment]
@BoneheadFraggle this card is no longer supported. Please switch to https://www.home-assistant.io/lovelace/alarm-panel/
